docs/our_work/ai-deep-dive.md

```diff
@@ -12,11 +12,11 @@ tags : ['BEST PRACTICE']
 
 ## Info
 
-This is a backup of the case study published [here](https://transform.england.nhs.uk/ai-lab/explore-all-resources/understand-ai/sharing-ai-skills-and-experience-through-deep-dive-workshops/) on the NHS England Transformation Directorate website.
+This is a backup of the case study published [here](https://digital.nhs.uk/services/ai-knowledge-repository/case-studies/sharing-ai-skills-and-experience-through-deep-dive-workshops) on the NHS England Transformation Directorate website.
 
 ### Case Study Overview
 
-The NHS AI Lab Skunkworks team provides public sector health and social care organisations with artificial intelligence (AI) support and technical expertise. The team of data scientists and AI project specialists has been helping others with their explorations with AI solutions for a range of problems, [from supporting hospital bed allocation to detecting CT scan anomalies](https://transform.england.nhs.uk/ai-lab/ai-lab-programmes/skunkworks/ai-skunkworks-projects/).
+The NHS AI Lab Skunkworks team provides public sector health and social care organisations with artificial intelligence (AI) support and technical expertise. The team of data scientists and AI project specialists has been helping others with their explorations with AI solutions for a range of problems, from supporting hospital bed allocation to detecting CT scan anomalies.
```

Comment:
I wonder that this starts an annoying precedent but is it worth using the past tense here?

Reply:
```suggestion
The NHS AI Lab Skunkworks team provided public sector health and social care organisations with artificial intelligence (AI) support and technical expertise. The team of data scientists and AI project specialists has helped others with their explorations with AI solutions for a range of problems, from supporting hospital bed allocation to detecting CT scan anomalies.
```